edenai_apis/apis/api4ai/api4ai_api.py

```diff
@@ -244,14 +244,16 @@ def image__anonymization(
         return result
 
     def image__logo_detection(
-        self, file: str, file_url: str = ""
+        self, file: str, file_url: str = "", model: str = None
     ) -> ResponseType[LogoDetectionDataClass]:
         file_ = open(file, "rb")
         payload = {
             "image": file_,
         }
         # Get response
-        response = requests.post(self.urls["logo_detection"], files=payload)
+        response = requests.post(
+            self.urls["logo_detection"].format(model=model), files=payload
+        )
```

Comment:
Like the way you did it with format and declaring the endpoints with the ...{model}.., stylé !